### PR TITLE
ci(deps): configure Dependabot to read from GH packages

### PR DESCRIPTION
**Pull request by ebaron**
I'm trying to get Dependabot to update cryostat-core. I followed this documentation: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/configuring-access-to-private-registries-for-dependabot

Related to: #56

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 version: 2
+registries:
+  maven-github:
+    type: maven-repository
+    url: https://maven.pkg.github.com/cryostatio/cryostat-core
+    token: ${{secrets.DEPENDABOT_READ_GHCR}}
```
Comment (tthvo) [inline .github/dependabot.yml:6]: Reading from this https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#maven-repository

> The maven-repository type supports username and password.

wondering if token still works?

Comment (ebaron) [reply .github/dependabot.yml]: Good catch! I've changed it.

**Review by tthvo — APPROVED** [review]
Looks good to me!

**Comment by ebaron** [comment]
@tthvo success! #77

**Comment by tthvo** [comment]
Woahh very niceee!!!

**Pull request merged**